tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -16,15 +16,17 @@ readonly script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
 readonly prefix="/opt/kata"
 readonly repo_root_dir="$(cd "${script_dir}/../../../.." && pwd)"
+readonly static_build_dir="$(cd "${script_dir}/../../static-build" && pwd)"
```

Comment:
It took me some mental effort to ensure that `${repo_root_dir}/tools/packaging/static-build` was the same as `${script_dir}/../../static-build`.  But this is way more on us than on you, @converge. :-)